### ISSUE TITLE
Some buttons don't work

### ISSUE THREAD

**Issue by anon88391** [HamletDuFromage/4chan_Mass_Quotes#33]
I select the 'sage' button in the quick reply box, but the thread gets bumped anyway.
The 'anonymize file hash' button in the QR box doesn't seem to do anything either. The toggle inside the addon settings for this does work, however.
The 'spoof filename' button in the QR box doesn't do anything for me either.
The remaining buttons seem to work as intended.

All of the above was the same in previous version but I was too lazy to open an issue about it.

**Comment by soyware** [comment]
sage works for me on firefox/chrome with/without 4chanx, buttons indeed don't have the logic for 4chanx, ill see if i can come up with something

**Comment by soyware** [comment]
indeed, seems like 4chanx clears email field after posting

**Comment by anon88391** [comment]
Now after I recently migrated to linux, changing the hash through the addon settings doesn't work either for me, in firefox or any chromium browser tested. Previously enabling it at least changed the files I posted even if the QR box button was broken, but now I can't flood threads with same one soyjak image, getting the duplicate file error. 
I tried really hard to get it working on various browsers as it felt odd to me that an OS change would cause this, but it's the same on all of them.
I tried to do some basic debugging to get some info, but I quit as I'm lazy and hate JS.

Nevermind, just quickly tested on my Windows laptop, doesn't work on it anymore either so I have no idea what happened - some 4chan or 4chanX code update?

**Comment by soyware** [comment]
yeah i noticed "+1" color change isnt enough in some situations, i changed it to +10 seems to work

**Comment by anon88391** [comment]
Doesn't work with gifs or webms for me either though

**Comment by anon88391** [comment]
Well I figured out what the issue is in my case. I've been posting files through the "post from URL" button, which seems doesn't trigger the 'change' event. Selecting it through the OS file picker works fine.
